catroid/src/main/res/values-ar/strings.xml

```diff
@@ -1447,4 +1447,13 @@
   <string name="crash_dialog_always_send">دائماً أرسل</string>
   <string name="crash_dialog_send">أرسل</string>
   <!--  -->
+  <!--  -->
+  <string name="Byte_short">بايت</string>
+  <string name="kiloByte_short">كيلوبايت</string>
```

Comment:
Please write kilo in upper case for consistencies sake.
Same goes for values/strings.xml